packages/explorer/src/components/ui/Button.tsx

```diff
@@ -0,0 +1,44 @@
+import { type VariantProps, cva } from "class-variance-authority";
+import * as React from "react";
+import { Slot } from "@radix-ui/react-slot";
+import { cn } from "../../lib/utils";
+
+const buttonVariants = cva(
+  // eslint-disable-next-line max-len
+  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",
```

Comment:
no change expected here, but just wanted to mention a thing I've been doing lately to make this a bit more readable is splitting these up into multiple strings, where each string groups properties by use (e.g. layout things vs text formatting vs state modifiers):

```suggestion
  "inline-flex items-center justify-center rounded-md",
  "whitespace-nowrap text-sm font-medium",
  "ring-offset-background transition-colors",
  "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2",
  "disabled:pointer-events-none disabled:opacity-50",
```

Reply:
Yeah, that's nicer, agreed. It would make for a nice prettier plugin